packages/@coorpacademy-app-review/nyc.config.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
     'src/actions/test',
```

Comment:
on peut effacer`'**/**.test.ts'` et `'src/actions/test' `au faveur de:
```suggestion
    '**/test/**',
```